src/ui/layout/RecentObjectsListItem.vue

```diff
@@ -54,6 +54,7 @@
     <div class="c-recentobjects-listitem__target-button">
         <button
             class="c-icon-button icon-target"
+            :aria-label="`Open and scroll to ${domainObject.name}`"
```

Comment:
back to our `id` conversation, is it possible to reference domainObject.name as an id and use aria-labelledby to compute this aria-label?

Reply:
`domainObject.name` is actually not unique, so wouldn't make a very good id, but I wonder if we can use the `navigationPath` instead as an id and reference that? we could then use aria-labelledby.

lemme play around with that

Reply:
So we can use something like `recent-object:${navigationPath}` to make a unique id for these list items, but we would still be missing a meaningful, unique a11y label to label it with. When you get some time let's chat about and revisit this